benchmarks/dynamo/common.py

```diff
@@ -1557,6 +1586,10 @@ def run(runner, args, original_dir=None):
                 if args.training
                 else CI_SKIP_INDCUTOR_INFERENCE
             )
+    if args.ddp:
+        assert args.accuracy, "TODO: hook DDP support to --speedup mode"
```

Comment:
Can you provide a more instructive error message here? not clear, what does args.accuracy + arg.ddp have to do with --speedup mode?